tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -2258,16 +2260,13 @@ def test_not_has_filter(self):
                     "transaction",
                     "p50()",
                 ],
+                # Doing !has on the metrics dataset doesn't really make sense
```

Comment:
This was the comment on the next query. Since we're completely disallowing `!has`, I had to change this test.